docs/user_guide/authenticate.md

```diff
@@ -1,11 +1,139 @@
+
+## Login using a `.netrc`
+
+### Creating a `.netrc` file
+
+#### Using `earthaccess.login` to create a `.netrc` file
+
+You can use `earthaccess.login` to create a `.netrc` for you.
+```
+earthaccess.login(persist=True)
+```
+You will then be prompted for your Earthdata Login username and password.  A `.netrc` (or `_netrc`) file will be created automatically.
+
+#### Manually creating a `.netrc` file for Earthdata Login Credentials
+
+=== "MacOS"
+    Type the following on your command line, replacing `<username>` and `<password>` with your
+    Earthdata Login credentials.
+    
+    ```
+    echo "machine urs.earthdata.nasa.gov login <username> password <password>" >> $HOME/.netrc
+    chmod 600 $HOME/.netrc
+    ```
+
+=== "Linux"
+    Type the following on your command line, replacing `<username>` and `<password>` with your
+    Earthdata Login credentials.
+    
+    ```
+    echo "machine urs.earthdata.nasa.gov login <username> password <password>" >> $HOME/.netrc
+    chmod 600 $HOME/.netrc
+    ```
+
+=== "Windows"
+    In a `CMD` session, create a `%HOME%` environment variable.  The following line
+    creates `%HOME%` from the path in `%USERPROFILE%`, which looks something like
+    `C:\Users\"username"`.
+    ```
+    setx HOME %USERPROFILE%
+    ```
+    You now need to create a `_netrc` file in `%HOME%`.
+    ```
+    echo "machine urs.earthdata.nasa.gov login <username> password <password>" >> %HOME%\_netrc
+    ```
+
+## Login using environment variables
+
+Alternatively, Earthdata Login credentials can be created as environment variables EARTHDATA_USERNAME and EARTHDATA_PASSWORD.
+
+=== "MacOS"
+    If you want to set the environment variables for the current shell session, type the following on the command line.
+    ```
+    export EARTHDATA_USERNAME="username"
+    export EARTHDATA_PASSWORD="password"
+    ```
+    If you want to set these environmental variables permanently, add these two lines to your `.profile` file.
+    
+
+=== "Linux"
+    If you want to set the environment variables for the current shell session, type the following on the command line.
+    ```
+    export EARTHDATA_USERNAME="username"
+    export EARTHDATA_PASSWORD="password"
+    ```
+    If you want to set these environmental variables permanently, add these two lines to your `.bashrc` file.
+
+=== "Windows"
+    To set the environment variables for the current `CMD` session, type the following:
+    ```
+    setx EARTHDATA_USERNAME "username"
+    setx EARTHDATA_PASSWORD "password"
+    ```
+    To set these environmental variables permanantly  
+    1. Open the start menu.  
+    2. Search for the "Advanced System Settings" control panel and click on it.  
+    3. Click on the "Environment Variables" button toward the bottom of the screen.  
+    4. Follow the prompts to add the variable to the user table.  
+
+
+## Accessing different endpoints
+
+### Earthdata User Acceptance Testing (UAT) environment
```

Comment:
```suggestion
### Earthdata User Acceptance Testing (UAT) endpoint
```

We use three terms in overlapping ways right now: "System", "environment", and "endpoint". I think since we use the term "system" for the API we should try and only use "system" in the docs. At the very least, we should eliminate the use of "environment"?